src/quo2/components/list_items/channel.cljs

```diff
@@ -59,6 +61,10 @@
                 (not muted?)
                 (not (pos? (int mentions-count)))
                 unread-messages?)
-       [quo2.icons/icon :i/channel-notification
-        {:size     20
-         :no-color true}])]]])
+       [rn/view
+        {:style {:width            8
```

Comment:
Nothing major, but you could extract the styles to a new `style.cljs` namespace.

Reply:
Sure!